friends/src/Components/Friends.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Loader from 'react-loader-spinner';
+import { getFriends } from '../actions';
+import Friend from './Friend'
+
+class Friends extends React.Component {
+    constructor() {
+        super();
+    }
+
+    componentDidMount() {
+        this.props.getFriends();
+    }
+    // state = {
+    //     newFriend: ''
+    // }
+
+    // handleChanges = e =>{
+    //     this.setState({newFriend: e.target.value});
+    // };
+
+    // addMember = e => {
+    //     e.preventDefault();
+    //     this.props.addMember(this.state.newFriend);
+    //     this.setState({ newFriend: ''})
+    // };
+
+    render() {
+        console.log(this.props.friends)
+        return (
+            <div className='friends'>
+                <h1>this is the friends page</h1>
+                {/* {this.props.friends.map(friend =>{
+                        return <Friend key = {friend.name} friend = {friend}/>
+                    }
+                )} */}
+            </div>
+        )
+    }
+}
+
+const mapStateToProps = state => ({
+    friends: state.friends,
+    error: state.error,
+    isLoading: state.isLoading
```

Comment:
When you are using `combineReducer` (which is usually when you have more than one reducer), then you'll have to specify which reducer you are getting properties from in the `mapStateToProps` function. Here, it would be:
```
const mapStateToProps = state => ({
    friends: state.friendReducer.friends,
    error: state.friendReducer.error,
    isLoading: state.friendReducer.isLoading
})
```

Adding `friendReducer` as shown above will fix your issue with the `friend` objects not displaying.